processor/main.py

```diff
@@ -36,14 +37,110 @@ def write_to_vdb(mapped: list) -> None:
             if w is not None:
                 w.close()
 
+def write_to_neo4j(events) -> None:
+        db = os.getenv("NEO4J_DB_HOST", "docker-neo4j-1")
+        db_port = os.getenv("NEO4J_DB_PORT", "7687")
+        url = "neo4j://" + db + ":" + db_port
+        username = "neo4j"
+        password = "password"
+        graph = Graph(url, auth=(username, password))
+        try:
+            graph.run("Match () Return 1 Limit 1")
+            print('successfully connected')
+        except Exception:
+            print('unsuccessful connection')
+        print("Writing to Neo4j at " + ":" + url + " ... " + str(len(events)))
+        person_list = []
+        events_list = []
+        attendance_list = []
+        try:
+            for record in events:
+                event = record.value
+                print("event: ", event)
+                status = ""
+                start = ""
+                end = ""
+                event_summary = ""
+                event_description = ""
+                event_recurring_id = ""
+                # get event information (meeting information)
+                event_id = event["id"]
+                if "status" in event:
+                    status = event["status"]
+                if "recurringEventId" in event:
+                    event_recurring_id = event["recurringEventId"]
+                if "start" in event:
+                    start = event["start"].get("dateTime", event["start"].get("date"))
+                if "end" in event:
+                    end = event["end"].get("dateTime", event["end"].get("date"))
+                if "summary" in event:
+                    event_summary = event["summary"]
+                if "description" in event:
+                    event_description = event["description"]
+                event_dict = {"id": event_id, "start": start, "end": end, "description": event_description,
```

Comment:
I would collapse lines 60 through 80 into a single line that leverages event.get(..) to create the dict and avoid any intermediate variables unless you explicitly need them. Short methods (<20-30 lines) are preferable for readability purposes